website/docs/r/frontdoor_custom_https_configuration.html.markdown

```diff
@@ -0,0 +1,155 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_frontdoor_custom_https_configuration"
+description: |-
+  Manages the Custom Https Configuration for an Azure Front Door Frontend Endpoint.
+---
+
+# azurerm_frontdoor_custom_https_configuration
+
+Manages the Custom Https Configuration for an Azure Front Door Frontend Endpoint..
+
+~> **NOTE:** Custom https configurations for a Front Door Frontened Endpoint be defined both within [the `azurerm_frontdoor` resource](frontdoor.html) via the `custom_https_configuration` block and by using a separate resource, as described in the following sections.
+
+-> **NOTE:** Defining custom https configurations using a separate `azurerm_frontdoor_custom_https_configuration` resource allows for parallel creation/update.
```

Comment:
shouldn't these disclaimers be added to the main frontdoor resource as well?

Reply:
👍